### DISCUSSION TITLE
79900BC2-FFE0-4CFA-AA65-B7B99CFA59C4

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#68] [General]
# 79900BC2-FFE0-4CFA-AA65-B7B99CFA59C4

TOI-715.02 is the second planet candidate of TOI-715, this planet is also projected to orbit within it's hosts habitable zone but farther out than TOI-715 b. The planet is nearly the same size as Earth (.02= 1.066 Re).

If confirmed it would be one of the smallest planets that orbits within the (conservative) habitable zone of its star.

NOTES:
- An existing TOI-715 addon must be installed in order for this addon to work
- This addon is NOT compatible with the Unconfirmed Exoplanets addon

https://celestia.mobi/addon?item=79900BC2-FFE0-4CFA-AA65-B7B99CFA59C4&lang=en

**Comment by exocritic5457** [comment]
I see what was attempted to be done here, a colder eyeball world representing that one impression of TRAPPIST-1 f. But the islands that are on the warm side seem to be way too evenly distributed, along with the rivers being nearly all straight. The islands are also unnaturally flat in comparison to the more rugged ice around the terminator. Overall, it's alright, good in what it represents, but it could use a bit more.
